ibm/service/power/resource_ibm_pi_volume_clone_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright IBM Corp. 2023, 2024 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	st "github.com/IBM-Cloud/power-go-client/clients/instance"
+	acc "github.com/IBM-Cloud/terraform-provider-ibm/ibm/acctest"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccIBMPIVolumeClone(t *testing.T) {
+	name := fmt.Sprintf("tf-pi-volume-clone-%d", acctest.RandIntRange(10, 100))
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { acc.TestAccPreCheck(t) },
+		Providers: acc.TestAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckIBMPIVolumeCloneConfig(name),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckIBMPIVolumeCloneExists("ibm_pi_volume_clone.power_volume_clone"),
+					resource.TestCheckResourceAttrSet("ibm_pi_volume_clone.power_volume_clone", "id"),
+					resource.TestCheckResourceAttrSet("ibm_pi_volume_clone.power_volume_clone", "volume_clone_status"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckIBMPIVolumeCloneExists(n string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+
+		rs, ok := s.RootModule().Resources[n]
+
+		if !ok {
+			return fmt.Errorf("Not found: %s", n)
+		}
+
+		if rs.Primary.ID == "" {
+			return errors.New("No Record ID is set")
+		}
+
+		sess, err := acc.TestAccProvider.Meta().(conns.ClientSession).IBMPISession()
+		if err != nil {
+			return err
+		}
+
+		ids, err := flex.IdParts(rs.Primary.ID)
+		if err != nil {
+			return err
+		}
+		cloudInstanceID, vcTaskID := ids[0], ids[1]
+		client := st.NewIBMPICloneVolumeClient(context.Background(), sess, cloudInstanceID)
+
+		_, err = client.Get(vcTaskID)
+		if err != nil {
+			return err
+		}
+		return nil
+	}
+}
+
+func testAccCheckIBMPIVolumeCloneConfig(name string) string {
+	return volumesCloneConfig(name, true) + fmt.Sprintf(`
+	resource "ibm_pi_volume_clone" "power_volume_clone" {
+		pi_cloud_instance_id   			= "%[1]s"
+		pi_volume_clone_name     		= "%[2]s"
+		pi_volume_ids 					= [ibm_pi_volume.power_volume[0].volume_id,ibm_pi_volume.power_volume[1].volume_id]
```

Comment:
```suggestion
		pi_volume_ids 					= ibm_pi_volume.power_volume.*.volume_id
```
This should work. Please test it out.